packages/db/src/network/test/mockProvider.ts

```diff
@@ -35,22 +39,22 @@ export const mockProvider = (options: {
             : undefined;
 
           return callback(null, {
-            jsonrpc,
+            jsonrpc: jsonrpc as JsonRpcIdentifier,
             id,
-            result
+            result: result as any
```

Comment:
Probably shouldn't be any (@spacesailor)

Reply:
Using `any` in tests is not a bad practice when you have to deal with multiple data. But if you want to avoid it you can use the right type.